modules/Makefile

```diff
@@ -15,6 +15,13 @@ obj-y += cas_disk/
 # line; invoke the kernel build system.
 else
 
+include ../tools/helpers.mk
+
+define remove-module
+	@if (lsmod | grep -q ${1}); then rmmod ${1}; \
+	else echo "WARNING: Module ${1} is not loaded"; fi
+endef
```

Comment:
Why not move it to the _helpers.mk_ like the other defines for _remove-file_ and _remove-directory_?

Reply:
This function isn't used in any other Makefile, so there is no need to move it to common code.